src/SprykerEco/Zed/Unzer/Business/Reader/UnzerReader.php

```diff
@@ -118,4 +113,66 @@ public function getPaymentUnzerTransactionByPaymentIdAndParticipantId(
                 ->findPaymentUnzerTransactionByPaymentIdAndParticipantId($paymentId, $transactionType, $participantId)
             ?? new PaymentUnzerTransactionTransfer();
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\CustomerTransfer $customerTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerCustomerTransfer|null
+     */
+    public function findUnzerCustomerTransferByCustomerTransfer(CustomerTransfer $customerTransfer): ?UnzerCustomerTransfer
+    {
+        return $this->unzerRepository->findUnzerCustomerByIdCustomer($customerTransfer->getIdCustomer());
```

Comment:
It's just a proxy method without any additional logic, I think it can be deleted.

Reply:
Just copied from branch. Should be fixed after merge ...